### ISSUE TITLE
are you sure about license?

### ISSUE THREAD

**Issue by tehKaiN** [AmigaPorts/ADL#1]
GPL is very viral - you need to submit all derivative works (game/demo itself) on GPL too. Perhaps something less restrictive but still ensuring back-contributing (LGPL, MPL) would be better?

See similar discussion we had in ACE: https://github.com/AmigaPorts/ACE/issues/43

**Comment by MBeijer** [comment]
Yeah, I have no idea which licence is best. Some of the code used should
fall under another license or could simply just be referenced by submodule.
